Scenarios.xcodeproj/project.pbxproj

```diff
@@ -363,13 +371,13 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Curious, were there any errors/warnings related to targeting 8.0?

Reply:
Nope, nothing on Xcode at least!

Reply:
May as well keep the same deployment target then?